### ISSUE TITLE
Peak picking on some spectra takes a very long time

### ISSUE THREAD

**Issue by lpatiny** [cheminfo-js/spectra-data#61]
Here is an example of jcamp file
[test.jdx.zip](https://github.com/cheminfo-js/spectra-data/files/826970/test.jdx.zip)

The process blocks at the level of
https://github.com/cheminfo/nmr-metadata/blob/17a39cfe2e0efb408a64561170139c87aa1135da/src/index.js#L102

**Comment by andcastillo** [comment]
Yes. There is a problem if you try to optimize too many peaks. Hopefully the Levenberg–Marquardt of Miguel is better than mine and we could optimize much more gaussians in a shorter time. By now, that problems has been solved in the new spectra-data releases by setting optimize to false by default. So, the solution is to update to a higher spectra-data version.

**Comment by andcastillo** [comment]
@lpatiny @maasencioh

**Comment by maasencioh** [comment]
@andcastillo this optimization it's related with this issue?

https://github.com/mljs/levenberg-marquardt/issues/4

**Comment by andcastillo** [comment]
Not really. What I mean is that I'm using my own LM implementation. It is a transcription from a matlab code.  Perhaps yours is better, but now, you need to make some improvements in order I can try to use it here. And the reason of the issue is that my parameters has many orders o magnitud of difference(x is 0.01 ppm range and intensity is 100000000), then, global options for them does not work properly.
